docs/quick-start.md

```diff
@@ -58,7 +58,7 @@ Open your `package.json` file and add the following scripts:
 {
   "scripts": {
     "dev": "bun --watch src/index.ts",
-    "build": "bun build src/index.ts",
+    "build": "bun build src/index.ts --target bun --outdir ./build",
```

Comment:
Can you please use `./dist` instead of `./build`? This corresponds to the default behavior of Bun:

![image](https://github.com/user-attachments/assets/19d0b6ab-b1ab-46a0-9c8d-6eef72a08069)

Reply:
i use `build` because it used here https://bun.sh/docs/bundler) But maybe dist will be better